exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -927,15 +927,11 @@ public void setMax(Object max) {
     @JsonProperty List<ParquetFileMetadata_v2> files;
     @JsonProperty List<String> directories;
     @JsonProperty String drillVersion;
-    @JsonProperty boolean isDateCorrect;
+    @JsonProperty int writerVersion;
```

Comment:
This property is Jackson-serialized. How does Jackson handle older files written without this version? According to this post (http://stackoverflow.com/questions/8320993/jackson-what-happens-if-a-property-is-missing), "Setter methods are only invoked for properties with explicit values." This means that older files without the writerVersion set won't call the function to deserialize the writer version, and the version will default to the newest version. I suspect that either A) I'm misunderstanding what this code does, or B) we have a backward-compatibility issue. Please explain which.